app/renderer-jsx/schema-form/components/array.jsx

```diff
@@ -35,13 +35,19 @@ class ArrayComponent extends React.Component {
   render() {
     const { schema, model, name, path, onChange, errors } = this.props;
     const arr = model || [];
-    let title = schema.title || name;
-    const description = utils.wrapDescription(schema.description);
 
+    let title = schema.title || name;
     if (title) {
       title = <h4>{title}</h4>;
     }
 
+    const description = utils.wrapDescription(schema.description);
+
+    let help;
```

Comment:
it can be replaced with `const help = schema.help && utils.wrapHelp(schema.help)` I think

Reply:
I just copied how title was handled above! 😄 

Changed.